foundations/html_css/flexbox/flexbox-growing-and-shrinking.md

```diff
@@ -76,4 +91,4 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-*   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
\ No newline at end of file
+- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
```

Comment:
```suggestion
* It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
```
**nitpick:** I believe these are supposed to stay as an asterisk according to the Layout Style Guide.

Reply:
Hey! As discussed with plans on adding a additional resources section I'll keep this commit open for the moment

Reply:
Check out the comment above about keeping the CSS Tricks link in the Assignment section. Then you can commit this suggestion for fixing the asterisk.

Reply:
@dm-murphy  As mentioned above I have committed this change!